backend/kernelCI_app/views/treeView.py

```diff
@@ -53,17 +53,17 @@ def get(self, request):
                 COUNT(DISTINCT CASE WHEN builds.valid IS NULL AND builds.id IS NOT NULL THEN builds.id END)
                     AS null_builds,
                 COUNT(DISTINCT builds.id) AS total_builds,
-                COUNT(CASE WHEN (tests.path <> 'boot' OR tests.path NOT LIKE 'boot.%%')
```

Comment:
We're not sure if there is a boot whose path is just `boot`, so I think we should keep `tests.path <> 'boot'` and use an `AND` instead of `OR`